CHANGELOG.md

```diff
@@ -44,10 +48,13 @@ BUG FIXES:
 
 **Enterprise LTS:** Vault Enterprise 1.19 is a [Long-Term Support (LTS)](https://developer.hashicorp.com/vault/docs/enterprise/lts) release.
 
+SECURITY:
+
+* auth/azure: Update plugin to v0.20.2. Login requires `resource_group_name`, `vm_name`, and `vmss_name` to match token claims [[HCSEC-2025-07](https://discuss.hashicorp.com/t/hcsec-2025-07-vault-s-azure-authentication-method-bound-location-restriction-could-be-bypassed-on-login/74716) [GH-30052](https://github.com/hashicorp/vault/pull/30052)].
+
 CHANGES:
 
 * UI: remove outdated and unneeded js string extensions [[GH-29834](https://github.com/hashicorp/vault/pull/29834)]
```

Comment:
Why are these Azure plugin update lines getting removed?

Reply:
they were moved to the security section above. Happy to keep them in the changes section as well if you prefer

Reply:
Ah! I see. I did not read carefully enough.